src/main/java/com/BrewMES/demo/model/Machine.java

```diff
@@ -26,7 +31,36 @@ public Machine(String ipAddress) {
     }
 
     public void controlMachine(Command command) {
-        throw new UnsupportedOperationException();
+            try {
+                //Create nodeID for Control Command.
+                NodeId cntrlCmd = new NodeId(6, "::Program:Cube.Command.CntrlCmd");
+
+                // Switch on the enum, writing different values to the machine.
+                switch(command){
+                    case RESET:
+                        connection.writeValue(cntrlCmd, DataValue.valueOnly(new Variant(1))).get();
```

Comment:
The switch cases probably have break statements.

Reply:
I changed it to enhanced switch.